Tests/test_image.py

```diff
@@ -688,6 +694,24 @@ def _make_new(
         _make_new(im, blank_p, ImagePalette.ImagePalette())
         _make_new(im, blank_pa, ImagePalette.ImagePalette())
 
+    @pytest.mark.parametrize("mode", image_modes)
+    def test_pixels_from_new_image_by_color(self, mode: ImageModeInfo) -> None:
+        if mode.name in ("BGR;15", "BGR;16"):
+            # These modes have fewer than 8 bits per band,
+            # so not all values can be round-tripped.
+            color = (16, 32, 49)
+        elif mode.num_bands == 1:
+            color = 123
+        else:
+            color = tuple(range(mode.num_bands))
+
+        im = Image.new(mode.name, (2, 2), color=color)
+
+        assert im.getpixel((0, 0)) == color
+        assert im.getpixel((0, 1)) == color
+        assert im.getpixel((1, 0)) == color
+        assert im.getpixel((1, 1)) == color
```

Comment:
```suggestion
        im = Image.new(mode.name, (1, 1), color)
        assert im.getpixel((0, 0)) == color
```
I don't think more than one pixel needs to be tested.